### DISCUSSION TITLE
License is locked Firefox

### DISCUSSION THREAD

**Discussion by EnnoKa** [aclap-dev/video-downloadhelper#1260] [License & Purchase requests]
### What's your issue?

My License is locked ,Firefox. no receiving emails to regularize.

email:  enno14@hotmail.de

### The 10 first digits (only 10) of your license number or email address

fe923d71f0

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Sorry you did not receive our email. I unlocked your license.
